packages/react/src/components/DateTimePicker/DateTimePicker.story.jsx

```diff
@@ -56,7 +56,6 @@ export const Default = () => {
     >
       <DateTimePicker
         id="datetimepicker"
-        dateTimeMask={text('dateTimeMask', 'YYYY-MM-DD HH:mm')}
```

Comment:
any reason it was removed ?

Reply:
we already have default value in DateTimePicker.

Reply:
![image](https://github.com/user-attachments/assets/049fb39a-fb98-4b94-94e3-53cabe06f0be)

Reply:
reverted.